config/environments/production.rb

```diff
@@ -12,4 +12,7 @@
   config.active_support.deprecation = :notify
   config.log_formatter = ::Logger::Formatter.new
   config.active_record.dump_schema_after_migration = false
+  #ignore email for now
```

Comment:
But how can you read my comment without the leading space? I noticed it after I made the commit, so I decided I'd just let it irritate hound.